packages/material-design-icons/package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this shouldn't have a package-lock file. You can just remove it.

Reply:
Deleted.